.github/workflows/format_code.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto-format code on every PR
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: autopep8
+        uses: peter-evans/autopep8@v1.2.1
```

Comment:
Does this format to `autopep8` instead of `black`?
If it is so, then don't change it. I actually wanted to shift to `autopep8`